lib/engine/round/operating.rb

```diff
@@ -409,7 +409,9 @@ def payout(revenue)
           payout_entity(player, per_share)
         end
         payout_entity(@share_pool, per_share, @current_entity)
-        change_share_price(:right)
+
+        jumps = [(revenue / current_entity.share_price.price).floor, @game.class::MAX_JUMPS]
```

Comment:
instead of this we should have a function that calculates the num jumps

Reply:
added although I am not sure if I did it right, probably not!